### DISCUSSION TITLE
What is `setClientNavigation()` for?

### DISCUSSION THREAD

**Discussion by frederikhors** [jycouet/kitql#92] [Q&A] [answered]
What is `setClientNavigation()` for?

**Comment by jycouet** [comment, marked as answer]
It's to distinguish between: client side navigation and direct hit
If you want more details I opened an issue here: https://github.com/sveltejs/kit/issues/4447

It's to run some optimizations on queries depending on this state.

**Comment by frederikhors** [comment]
Is `InitClientNavigation` disappeared in https://github.com/jycouet/kitql/tree/main/packages? After updating to `@kitql/all-in": "0.6.0`"?

**Reply by jycouet** [reply]
Yes, no need anymore!
All included in `all-in` 👍
